turbo.json

```
@@ -0,0 +1,20 @@
+{
+  "$schema": "https://turbo.build/schema.json",
+  "pipeline": {
+    "start": {
+      "cache": false,
+      "persistent": true
+    },
+    "build": {
+      "outputs": ["dist/**"]
+    },
+    "clean": {
+      "cache": false
+    },
+    "test": {
```

Needs to depend on `build`
```suggestion
    "test": {
      "dependsOn": ["^build"]
    }  
```